src/test/java/assignment1/Assignment1Test.java

```diff
@@ -0,0 +1,46 @@
+package assignment1;
+
+import assignment1.exceptions.InvalidArguementsException;
+import assignment1.services.CommandLineArguementsParser;
+import assignment1.services.ValidateArguements;
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.DisplayName;
+
+public class Assignment1Test {
+
+  @Test
+  @DisplayName("Exception thrown if name is not being passed first")
+  public void verifyNameNotBeingWrittenFirst() {
+    Assertions.assertThrows(InvalidArguementsException.class, () -> {
+      ValidateArguements.validateInputs(
+          new String[]{"-price", "20", "-name", "paneer", "-type", "raw"});
+    });
+  }
+
+  @Test
+  @DisplayName("Exception thrown if not enough arguements are passed")
+  public void verifyNotEnoughArguements() {
+    Assertions.assertThrows(InvalidArguementsException.class, () -> {
+      ValidateArguements.validateInputs(new String[]{"-name", "20"});
+    });
+  }
+
+  @Test
+  @DisplayName("SI Hashmap is being created properly")
+  public void verifyHashMapFormation() {
+    HashMap<String, String> map = new HashMap<String, String>();
```

Comment:
create a testcase to make sure only valid values is being passed to price and quantity as well